libs/utils/utils.js

```diff
@@ -365,16 +390,21 @@ export async function loadTemplate() {
 
 export async function loadBlock(block) {
   const name = block.classList[0];
-  const { miloLibs, codeRoot } = getConfig();
+  const { miloLibs, codeRoot, p13nBlocks } = getConfig();
```

Comment:
Curious about the "p13nBlocks" name. Just a reference for personalization?

Reply:
yep, just an abbreviation for personalization.  Similar to l10n = localization and a11y = accessibility